src/generator/AutoRest.Go.Tests/src/tests/generated/body-byte/bytegroupoperations.go

```diff
@@ -0,0 +1,287 @@
+package bytegroup
```

Comment:
I am confused whether to put the generated code in PR or not. I have included the generated tests in case if someone wants to refer to them but not sure what is the right way. Let me know if these should be removed.